crates/alerter/Cargo.toml

```diff
@@ -11,6 +11,7 @@ chrono = { workspace = true }
 clap = { workspace = true }
 observe = { path = "../observe" }
 model = { path = "../model" }
+number = { path = "../number" }
```

Comment:
I think this is the right spot.

We also have a "numeric-conversions" crate - I would actually suggest (not for this PR, but just as a follow-up cleanup) to collapse:
- The non-zero U256
- Various U256 extensions we have peppered throughout the code (`solver::*::u256`, `driver::*::u256`, `shared::conversions`, and I'm sure there are more)
- U256 serde `(Des|S)erializeAs` implementations

Reply:
**Edit**: looks like you already did some of my suggestions in this PR.